### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by b-buchkevych**
- [DEMO LINK](https://b-buchkevych.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://b-buchkevych.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,6 +11,52 @@
...
+
+    <form 
+      action="#"
+      method="post"
+      class="form small-form"
+      data-qa="small"
+    >
+      <label
+      for="small-search" 
+      >
```
Comment (tetiana-ivanchenko) [inline src/index.html:45]:
```suggestion
      <label
        for="small-search" 
      >
```

File: src/index.html [hunk 2]
```diff
@@ -11,6 +11,52 @@
...
+      </label>
+      <input
+        type="text"
+        name="small-search"
+        class="search small-search"
+        id="small-search"
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
+  
+    </form>
```
Comment (tetiana-ivanchenko) [inline src/index.html:60]:
```suggestion
      >
    </form>
```

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+body {
+  font-family: "Avenir", sans-serif;
+}
+
+.form {
+  position: relative;
+}
+
+.big-form {
```
Comment (tetiana-ivanchenko) [inline src/style.css:27]: please, follow BEM metosology for class names, also  follow recommendation from checklist, point 2

**Review by tetiana-ivanchenko — CHANGES_REQUESTED** [review]
fix comments and add test-report and demo links to the PR description

**Review by tetiana-ivanchenko — CHANGES_REQUESTED** [review]
fix all styles according to checklist, you don't need to have 3 classes like "form, form-small form--big", use only form for one, and modificator for second element
image

**Review by sTorba24 — CHANGES_REQUESTED** [review]
Fix previous comment please

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+  width: 19px;
+}
+
+.search {
+  font-family: inherit;
+  background: var(--background-color);
+  border: 1px solid var(--border-color);
+  box-shadow: 0 1px 8px var(--shadow-color) 0.1;
+  border-radius: 4px;
+  color: #3d4e61;
```
Comment (sTorba24) [inline src/style.css:49]: you have variable for it

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+  height: 11px;
+  width: 11px;
+}
+
+.search--small {
+  font-size: 14px;
+  padding-left: 33px;
+  height: 42px;
+  line-height: 19px;
+  width: 100%;
```
Comment (sTorba24) [inline src/style.css:76]: you have set it on .search

**Review by sTorba24 — APPROVED** [review]
GJ 🚀 Little comments left

**Pull request closed without merging**